### PR TITLE
[AMS] update AMS stable version to include Edge list api

### PR DESCRIPTION
**Pull request by giakas**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/202/Overall-Process-of-AME-Onboarding) for the workflow.
- [x] [Validation tools](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/84/Swagger-Validation-tools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/FAQ.md).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process]( https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/212/Swagger-PR-Review).

### CODE REVIEW

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json [hunk 1]
```diff
@@ -334,6 +334,61 @@
       },
       "type": "object"
     },
+    "ListEdgePoliciesInput": {
```
Comment (ravbhatnagar) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json:337]: Adding new properties to existing API version is classified as breaking change as per Azure API guideline

Comment (ravbhatnagar) [reply specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json:337]: Such change requires a new api version.

**Review by ravbhatnagar — COMMENTED** [review]
@giakas - few comments

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json (hunk 1 above)
Comment (giakas) [reply specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json:337]: @ravbhatnagar This is a new definition, not a new property, we are adding a new action to the entity, no new properties are added.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Review by allenjzhang — APPROVED** [review]

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json [hunk 2]
```diff
@@ -334,6 +334,61 @@
...
+          "description": "SAS token needed to interact with Event Hub."
+        }
+      },
+      "type": "object"
+    },
+    "EdgeUsageDataCollectionPolicy": {
+      "properties": {
+        "dataCollectionFrequency": {
+          "type": "string",
+          "description": "Data collection frequency."
```
Comment (allenjzhang) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Accounts.json:367]: nit. Should describe the expected format given it is string type. Same below as well.

**Pull request merged**